pennylane/shadows/transforms.py

```diff
@@ -33,13 +33,14 @@ def _replace_obs(tape, obs, *args, **kwargs):
                 f"Tape measurement must be {qml.measurements.Shadow!r}, got {o.return_type!r}"
             )
 
-    with qml.tape.QuantumTape() as new_tape:
+    with qml.queuing.AnnotatedQueue() as q:
         # queue everything from the old tape except the measurement processes
         for op in tape.operations:
             qml.apply(op)
 
         # queue the new observable
         obs(*args, **kwargs)
+    qscript = qml.tape.QuantumScript.from_queue(q)
```

Comment:
```suggestion
    qscript = qml.tape.QuantumScript(tape.operations, [obs(*args, **kwargs)])
```

Then again, here its saying obervable and not meausrementproceess, so maybe that won't work.